src/components/application_manager/rpc_plugins/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # --- Utils
+include("./helpers.cmake")
```

Comment:
@BSolonenko can you specify just `include("helpers.cmake")` ?

Reply:
@LuxoftAKutsan fixed in https://github.com/smartdevicelink/sdl_core/pull/2780/commits/d4ee98f8065da4e11821acbf9f6d8a36940424bd